### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by nelrafa103**

### CODE REVIEW

File: Chess_Chips.py [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
-
-    def Movement_In_L(self, value_x, value_y):
-        self.value_x = value_x
-        self.value_y = value_y
-        self.Walk_Horizontal(self.value_x)
-        self.Walk_Vertical(self.value_y)
-
-
-Chips_On_Game = ChipsOfGame(None,None,None)
+class chips_of_game(AuxiliarClass):
```
Comment (CristianUser) [inline Chess_Chips.py:11]: class names should be in PascalCase

File: Chess_Chips.py [hunk 2]
```diff
@@ -1,36 +1,48 @@
...
+            for x in values_of_chips[self.color][self.chip_type]:
+                if self.value == x:
+                    values_of_chips[self.color][self.chip_type][counter] = self.new_value
+                counter += 1
+
+
+#chips_on_game = chips_of_game(9, 9, 'red')
+
+
+class play(AuxiliarClass):
```
Comment (CristianUser) [inline Chess_Chips.py:29]: class names should be in PascalCase

File: Chess_Chips.py [hunk 3]
```diff
@@ -1,36 +1,48 @@
...
+        print(self.count)
+        print(self.color)
+        print(self.value_range)
+        print( self.rules_for_chips(self.chip_position,self.value_range)       )            #Reglas de las fichas si se mueve
+        self.cop_1 = int(self.chip_position[0])
+        self.cop_2 = self.chip_position[1]
+        print(self.cop_1,self.cop_2)
+      #  self.operating_system() #Limpia la consola cada cierto tiempo
+       # self.search_of_chips(self.entry_2) 
+
```
Comment (CristianUser) [inline Chess_Chips.py:48]: Please remove these spaces and let the line empty
```suggestion

```

File: Executable.py [hunk 4]
```diff
@@ -1,17 +1,21 @@
-from Chess_Board import Rules_
-from Chess_Board import Messages
-from Chess_Board import LETTERS_IN_BOARD
-from Chess_Board import VALUES_OF_THE_CHIPS
-from Chess_Chips import Chips_On_Game
-#Movements = ChipsOfGame()
-#Actions = {'Pawns': ChipsOfGame.Movement_Front()}
+from Chess_Chips import play
```
Comment (CristianUser) [inline Executable.py:1]: file names should be in snake_case

Comment (nelrafa103) [reply Executable.py:1]: Okay, im going to solve it right now,

Comment (nelrafa103) [reply Executable.py:1]: I think that im going to finish this week, jejej

File: Chess_Chips.py (hunk 3 above)
Comment (nelrafa103) [reply Chess_Chips.py:48]: What you mean?

**Comment by nelrafa103** [comment]
Do You recommend me to make another file where I will save everything that has to do with graphics? I was planning to do it in executale.py

**Comment by nelrafa103** [comment]
That's all for today

File: chess_board.py [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+                os.system('clear')
+            else:
+                os.system('cls')
+
+    def status(self):
+        print(values_of_chips)
+        print("Revision")
+
+
+auxiliar = AuxiliarClass()
```
Comment (CristianUser) [inline chess_board.py:164]: missing EOF empty line

File: executable.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
+from chess_chips import Play
```
Comment (CristianUser) [inline executable.py:1]: instead of `executable.py` change it for `main.py`

**Pull request merged**